src/main/java/strategies/impl/AllStrategyImpl.java

```diff
@@ -0,0 +1,33 @@
+package strategies.impl;
+
+import strategies.SearchStrategy;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class AllStrategyImpl implements SearchStrategy {
+
+    @Override
+
+    public List<String> search(List<String> list, Map<String, ArrayList<Integer>> searchMap, String query) {
+        String[] words = queryWords(query);
+        List<String> found = new ArrayList<>();
+
+        for (int index : searchMap.getOrDefault(words[0], new ArrayList<>())) {
+            boolean temporaryIndex = false;
+
+            for (String word : words) {
+                if (!list.get(index).toUpperCase().contains(word)) {
```

Comment:
You are using the "toUpperCase()" method in 5 places, please fix that.

![image](https://user-images.githubusercontent.com/27965136/147774388-71504cbb-df71-41c2-9502-2a8b24d87eac.png)